src/components/app/landing/login/LoginModal.tsx

```diff
@@ -9,16 +9,12 @@ interface Props {
 }
 
 const LoginModal = (props: Props) => {
-  const [formIsValid, setFormIsValid] = useState<boolean>(false);
-  const [enteredEmail, setEnteredEmail] = useState<string>('');
-  const [enteredPassword, setEnteredPassword] = useState<string>('');
+  const [form] = Form.useForm();
+  const [, forceUpdate] = useState<object>({});
```

Comment:
don't use state here 
make it a simple function 
`forceUpdate` use better var name

Reply:
@aritrasen12345 What is the use of this state?